content/blog/2021-04-15-april-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,187 @@
+﻿---
+title: April '21 Heartbeat
+date: 2021-04-15
+description: |
+  Monthly updates are here! Loads from the Community, use cases, videos, 
+  5 new hires and more!
```

Comment:
💅 
```suggestion
  5 new hires, and more!
```

Reply:
You guys are really comma happy. Ok.  I will submit.

Reply:
Also added comment after member in long version for consistency.  😉

Reply:
Haha. I usually avoid going into stylistic suggestions but I thought "5 new hired and more" sounded a bit like "more than 5 new hires".